dell/xps/15-9500/xps-common.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, ... }:
+let
+  thermald-conf = ./thermald-conf.xml;
+in
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+  ];
+
+  # Boot loader
+  boot.kernelParams = lib.mkDefault [ "acpi_rev_override" ];
+
+  # This will save you money and possibly your life!
+  services.thermald.enable = lib.mkDefault true;
+
+  # Thermald doesn't have a default config for the 9500 yet, the one in this repo was generated with dptfxtract-static:
+  services.thermald.configFile = lib.mkDefault thermald-conf;
```

Comment:
Is there a one-liner on how to generate this, nver used `dptfxtract-static`.

Reply:
I pulled it from here: https://github.com/intel/dptfxtract , the static version is linked statically and ran on my system. It generated /etc/thermald/thermald-conf.xml files and I brought over the auto selected one. I suppose it might be good to package the tool, as it would be as simple as `sudo ./dptfxtract-static` for folks to generate their own. I was also planning on putting a ticket in at https://github.com/intel/thermal_daemon for the new system (once that's done it should be possible to deprecate these parts of the config).

Reply:
Ok. Maybe put a link to the github repo for now.